pyproject.toml

```diff
@@ -95,6 +95,7 @@ torchvision = [
 
 [tool.poetry.extras]
 tesserocr = ["tesserocr"]
+paddleocr = ["paddlepaddle", "paddleocr"]
```

Comment:
For adding extras, these should also be in the main dependencies, with the `optional=true` flag

Reply:
<img width="1051" alt="Screenshot 2024-11-20 at 7 49 18 PM" src="https://github.com/user-attachments/assets/32cba18f-cc03-4d7e-8bfd-79849fbdaeb2">

Hey, I am getting this numpy version issue however, in my environment I have:
- numpy == 1.26.4
- deepsearch-glm == 0.26.1 
- paddlepaddle == 2.6.2
- paddleocr == 2.9.1

the library seems to be working fine, all the tests are getting passed as well. Can you please help me in resolving this?

Reply:
- which python version?
- can you please try to rebase with the latest main branch? we just merged something about numpy conflicts as well

Reply:
These are the lines I added in pyproject.toml to add the necessary libraries for paddle_ocr:
<img width="660" alt="Screenshot 2024-11-20 at 11 06 42 PM" src="https://github.com/user-attachments/assets/716504cd-73da-4a91-9244-90d983539341">
This is the error I am getting: 
<img width="764" alt="Screenshot 2024-11-20 at 11 08 43 PM" src="https://github.com/user-attachments/assets/511ca295-af50-46fd-9568-e0955dcfa56e">



I have rebased with the latest main branch however i am still facing the numpy version conflict between paddleocr and deepsearch-glm.